python/snewpy/scripts/snewpy_to_marley.py

```diff
@@ -0,0 +1,95 @@
+
+import logging
+import os
+import re
+import tarfile
+import io
+import json
+
+import numpy as np
+import pandas as pd
+from astropy import units as u
+from warnings import warn
```

Comment:
```suggestion
import re
import tarfile
import io
import json

import numpy as np
from astropy import units as u
```
Remove unused imports.